modules/schema/src/main/java/org/apache/ignite/internal/schema/NumericTypeUtils.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.schema;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import org.apache.ignite.schema.ColumnType;
+
+/**
+ * Numeric types utility class.
+ * */
+public class NumericTypeUtils {
+    /**
+     * Calculates byte size for BigInteger value.
+     * */
+    public static int sizeInBytes(BigInteger val) {
+        return val.bitLength() / 8 + 1;
```

Comment:
8-bit length value will return 2. Is it ok?
```suggestion
        return (val.bitLength() + 7) / 8;
```

Reply:
We use the same expression in Row/Columns classes for null-map size calculation.
I don't think this method is useful, maybe smth like next worth to be added to the common IgniteUtils:
`int bytesForBits(int numberOfBits)`

Reply:
BigInteger returns the bits size without regard to the sign. So, 8 bits is already a number exceeding Byte.MAX_VALUE. Here is an example of a number that the suggested function does not work on.

BigInteger integer = BigInteger.valueOf(Byte.MAX_VALUE + 1);
System.out.println(integer.toByteArray().length); // 2
System.out.println(integer.bitLength()); // 8
System.out.println(integer.bitLength() / 8 + 1); // 2
System.out.println((integer.bitLength() + 7) / 8); // 1 (!)

Reply:
Got it!